# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/HelpersSkia.h
# Commit: 9bcf4283ca00
# Full Hash: 9bcf4283ca003f8572392b8f111361c12eecde3d
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2019-03-31 09:00:52
# Regressor Bug: 1539026
# File Overlap Count: 1
# Description:
#   Bug 1539026 - Skia m74 Moz2D fixes. r=rhunt
# ==============================================================================

diff -r 01b0009b3f67 -r 9bcf4283ca00 gfx/2d/HelpersSkia.h
--- a/gfx/2d/HelpersSkia.h	Tue Mar 26 11:06:47 2019 -0400
+++ b/gfx/2d/HelpersSkia.h	Tue Mar 26 11:06:56 2019 -0400
@@ -280,18 +280,18 @@
   return SkShader::kClamp_TileMode;
 }
 
-static inline SkPaint::Hinting GfxHintingToSkiaHinting(FontHinting aHinting) {
+static inline SkFontHinting GfxHintingToSkiaHinting(FontHinting aHinting) {
   switch (aHinting) {
     case FontHinting::NONE:
-      return SkPaint::kNo_Hinting;
+      return kNo_SkFontHinting;
     case FontHinting::LIGHT:
-      return SkPaint::kSlight_Hinting;
+      return kSlight_SkFontHinting;
     case FontHinting::NORMAL:
-      return SkPaint::kNormal_Hinting;
+      return kNormal_SkFontHinting;
     case FontHinting::FULL:
-      return SkPaint::kFull_Hinting;
+      return kFull_SkFontHinting;
   }
-  return SkPaint::kNormal_Hinting;
+  return kNormal_SkFontHinting;
 }
 
 static inline FillRule GetFillRule(SkPath::FillType aFillType) {